Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,50 @@
-/*  Start coding below for POSITION THE CUBES TASK*/
+/*  Start coding below for POSITION THE CUBES TASK*/
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.parent{
+    height: 400px;
+    width: 400px;
+    background-color: #675BC6;
+}
+
+.child{
+    height: 100px;
+    width: 100px;
+}
+
+.one{
+    position: fixed;
+    border: 2px solid white;
+    top:0px;
+    left:0px;
+    background-color: #60B177;
+}
+

```

Comment:
The 'left' and 'right' properties should not be used together in the '.two' class. Choose one positioning property for proper alignment.